wgpu-types/src/lib.rs

```diff
@@ -234,7 +234,7 @@ bitflags::bitflags! {
         /// - All
         ///
         /// This is a native only feature.
-        const MAPPABLE_PRIMARY_BUFFERS = 0x0000_0000_0001_0000;
```

Comment:
Watch these, all native features are in a different space. Native ones are bits 16-63, webgpu ones are 0-15.

Reply:
Ah, good catch. Updated.